app/uk/gov/hmrc/exports/services/mapping/goodsshipment/consignment/DepartureTransportMeansBuilder.scala

```diff
@@ -38,9 +34,9 @@ class DepartureTransportMeansBuilder @Inject()() {
   ): Consignment.DepartureTransportMeans = {
     val departureTransportMeans = new DepartureTransportMeans()
 
-    inlandModeOfTransportCode.flatMap(_.inlandModeOfTransportCode).foreach { value =>
+    inlandModeOfTransportCode.flatMap(_.inlandModeOfTransportCode).foreach { inlandModeOfTransportCode =>
```

Comment:
I can't work out why this has changed.  `InlandModeOfTransportCode` hasn't changed has it?

Reply:
Ignore me.